tests/test_mpi.py

```diff
@@ -1491,6 +1491,35 @@ def test_staggering(self):
         assert np.isclose(norm(uxx), 80001.0304, rtol=1.e-4)
         assert np.isclose(norm(uxy), 61427.853, rtol=1.e-4)
 
+    @skipif('nompi')
```

Comment:
nitpicking: this one is probably unnecessary as already present at the top of the file

Reply:
Ah, sure, lack of checking after the copy and paste sorry.